packages/lisk-transaction-pool/src/transaction_pool.ts

```diff
@@ -0,0 +1,109 @@
+/*
+ * Copyright © 2018 Lisk Foundation
+ *
+ * See the LICENSE file at the top-level directory of this distribution
+ * for licensing information.
+ *
+ * Unless otherwise agreed in a custom licensing agreement with the Lisk Foundation,
+ * no part of this software, including this file, may be copied, modified,
+ * propagated, or distributed except according to the terms contained in the
+ * LICENSE file.
+ *
+ * Removal or modification of this copyright notice is prohibited.
+ *
+ */
+import { Queue } from './queue';
+
+	// tslint:disable
```

Comment:
this would disable all =0

Reply:
There are many TSlint related issues because the functionality isn't implemented. Therefore, I've disabled it, once it's implemented, we can remove this line.